NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs

```diff
@@ -29,7 +29,6 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
             View view = inflater.Inflate(Resource.Layout.consent_settings_page_body, container, false);
 
             RelativeLayout RelativeLayout4 = view.FindViewById<RelativeLayout>(Resource.Id.consent_paragraph_hvordan_accepterer);
-            TextView consentFourParagraph = RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_title);
```

Comment:
Have you tested that this is not needed?

Reply:
yes, IDE even marks an error here as Resource.Id.consent_page_title does not exist anymore

Reply:
Hmmm... exciting... I didn't show any error when I implemented it :)

Reply:
wrong IDE ;) i am using Rider